modules/lang-painless/src/main/java/org/elasticsearch/painless/ir/ClassNode.java

```diff
@@ -152,34 +169,25 @@ public ScriptRoot getScriptRoot() {
         constructor.returnValue();
         constructor.endMethod();
 
+        if (clinitNode.getBlockNode().getStatementsNodes().isEmpty() == false) {
+            ReturnNode returnNode = new ReturnNode();
+            returnNode.setLocation(new Location("internal$clinit$return", 0));
+            clinitNode.getBlockNode().addStatementNode(returnNode);
```

Comment:
I changed this to do no modifications of the ir tree from write. I just do the method writer for clinit and the return statement in the ClassNode. I'll consider better ways to do this, but for now this makes the most sense because it's the simplest change.